app/filters/sanitize_filter.rb

```diff
@@ -27,9 +26,14 @@ def strip_event_handlers(parser)
     parser.search("*").each do |elem|
       elem.attributes.each do |name, attr|
         # XSS fix
-        elem.remove_attribute(name) if attr.value && attr.value.downcase.gsub(/[\\]*/, '') =~ /^[\s]*javascript\:/
+        if attr.value &&
+           attr.value.downcase.gsub(/[\\]*/, "") =~ /^[\s]*javascript\:/
```

Comment:
Use 4 (not 3) spaces for indenting a condition in an `if` statement spanning multiple lines.